### ISSUE TITLE
In lines 201 and 202 of the code, both alpha and alpha_bar use the same set of parameters.

### ISSUE THREAD

**Issue by NumnumnumM** [luost26/diffab#26]
https://github.com/luost26/diffab/blob/c3e2966601bf8025025ab87717b31b08fdd4834e/diffab/modules/diffusion/transition.py#L201C9-L201C61

**Comment by tsjain** [comment]
Yes. I had the same question about the calculation of the sequence posterior. 
Can the authors please comment?
